web/src/js/backends/static.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * This backend uses the REST API only to host static instances,
+ * without any Websocket connection.
+ */
+import { fetchApi } from "../utils"
+
+const CMD_RESET = 'reset'
+
+export default class StaticBackend {
+   constructor(store) {
+       this.activeFetches = {}
+       this.store = store
+       this.onOpen()
+   }
+
+   onOpen() {
+        this.fetchData("settings")
+        this.fetchData("flows")
+        this.fetchData("events")
+        this.fetchData("options")
+   }
+
+   fetchData(resource) {
+       let queue = []
+       this.activeFetches[resource] = queue
+       fetchApi(`/${resource}`)
+           .then(res => res.json())
+           .then(json => {
+               if (this.activeFetches[resource] === queue)
+                   this.receive(resource, json)
+           })
+   }
+
+   onMessage(msg) {
```

Comment:
For now, the only way to load flows into the static view is to use the `open` option in the `FileMenu`, and then we could refresh the page to see the flows. So, we need the whole `activeFetches` thing to update the flows view, is that unnecessary if we have the `flow` -> static converter?